### ISSUE TITLE
Feature request:  Log formatting

### ISSUE THREAD

**Issue by sslupsky** [drewfish/arduino-ZeroRegs#6]
It would be helpful if a timestamp could be added and the output could be formatted as logfmt.

https://www.brandur.org/logfmt

ie:  `<timestamp> <key1>=<value1> <key2>=<value2> ...`

example:

`[1234] level=info msg="<insert zeroregs info here">"`

**Comment by drewfish** [comment]
This is an interesting idea. It might be possible with the current version of ZeroRegs, since `struct ZeroRegOptions` has a `Stream&` member, which you could specify as your own custom implementation which wraps the message as you suggest. I'll play around to see if that might work.

**Comment by drewfish** [comment]
Yeah I think you can do this yourself by creating a `Stream` which wraps the output. Check out PR #8, which might in fact be what you want.

**Comment by sslupsky** [comment]
Thanks for the example.  I haven't had a chance yet to test it out but I should have some time next week to try it.
